src/v/resource_mgmt/cpu_profiler.h

```diff
@@ -70,12 +70,15 @@ class cpu_profiler : public ss::peering_sharded_service<cpu_profiler> {
 
     // Collects `shard_results()` for each shard in a node and returns
     // them as a vector.
-    ss::future<std::vector<shard_samples>>
-    results(std::optional<ss::shard_id> shard_id);
+    ss::future<std::vector<shard_samples>> results(
+      std::optional<ss::shard_id> shard_id,
+      std::optional<ss::lowres_clock::time_point> filter_before = std::nullopt);
 
     // Returns the samples and dropped samples from the shard this function
     // is called on.
```

Comment:
Missing doc for the new `filter_before` option.

Reply:
Updated doc comment to include `filter_before`.